src/libs/ReportUtils.js

```diff
@@ -1782,9 +1782,10 @@ function getTransactionReportName(reportAction) {
  * @param {Object} [reportAction={}] This can be either a report preview action or the IOU action
  * @param {Boolean} [shouldConsiderReceiptBeingScanned=false]
  * @param {Boolean} isPreviewMessageForParentChatReport
+ * @param {Boolean} shouldHidePayerName
  * @returns  {String}
  */
-function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceiptBeingScanned = false, isPreviewMessageForParentChatReport = false) {
+function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceiptBeingScanned = false, isPreviewMessageForParentChatReport = false, shouldHidePayerName = false) {
```

Comment:
we should avoid having such a number of parameters. Can you use one options object instead of four parameters? I don't know how time-consuming it can be

Reply:
I know, I would do that for all the functions here with more than 3 params (there's a lot of such functions here). 😅 But maybe it should be done in some major refactor?

Reply:
Maybe just these functions that You extends? That avoids generating new technical debt